### DISCUSSION TITLE
How do I properly scale a tilemap

### DISCUSSION THREAD

**Discussion by SarahCosmosys** [HaxeFlixel/flixel#2340] [Q&A]
I'm trying to make a simple pixel art game. My tiles are 16x16 px, I want to scale the tilemap by 4. How do I properly scale it updating collision?

**Comment by RichardBray** [comment]
That's a good question, I'm not really sure. Maybe you'll have to scale the collisions as well 🤷‍♂️ 

Then again it depends on what you want to do. There is a way to zoom the whole game in which means the collisions zoom as well.

**Reply by Geokureli** [reply]
you may need to set the tilemap's tile size to 64x64, but you can probably save a lot of headache by scaling the tileset image up in an art program and treating it like a 64x64 tileset
